assignments/arrays.js

```diff
@@ -63,34 +63,57 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make}  ${inventory[32].car_model}` );
 
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = `The last car make is  ${inventory[49].car_make} and model ${inventory[49].car_make} `;
+console.log(lastCar);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+
+for (let i = 0; i < inventory.length; i++) {
+    carModels.push(inventory[i].car_model)
+   }
+   
+console.log(carModels.sort());
 
 // ==== Challenge 4 ====
```

Comment:
Your answer is right, but the question only asked for the years. You didn't need to sort them.  Kuddos for going the extra step but always be sure to double check your answering what's being asked :)

Reply:
fixed